### PR TITLE
fixed poetry and node versions in Dockerfile

### PR DESCRIPTION
**Pull request by ldt**
Hi
Thanks for your great sample of integrating nginx Unit, fastapi and vue.js in a single docker image.
In order to build it, I had to fix the node version to v16. I also took the liberty to fix the version of poetry

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 ENV POETRY_VIRTUALENVS_IN_PROJECT=true
 
 COPY backend/pyproject.toml backend/poetry.lock ./
-RUN poetry install --no-dev
 COPY backend/app/ app/
+RUN poetry install --only main
 
 
-FROM node:lts as build-frontend
+FROM node:16.20.2-alpine3.18 as build-frontend
```
Comment (mattmess1221) [inline Dockerfile:14]: Any reason to use alpine for the build layer?

**Comment by mattmess1221** [comment]
Thanks for bringing this up.
239bcc7e29be6d18731a17350c992b93fe4517c1

File: Dockerfile (hunk 1 above)
Comment (ldt) [reply Dockerfile:14]: I wanted to reduce the size of the docker image

Comment (ldt) [reply Dockerfile:14]: but just fixing the version of node would suffice if you have any doubt about using the alpine version

**Pull request closed without merging**